specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json

```diff
@@ -1368,495 +1368,2776 @@
+        },
+        "ipRangeFilter": {
+          "description": "Cosmos DB Firewall Support: This value specifies the set of IP addresses or IP address ranges in CIDR form to be included as the allowed list of client IPs for a given database account. IP addresses/ranges must be comma separated and must not contain any spaces.",
+          "$ref": "#/definitions/IPRangeFilter"
+        },
+        "isVirtualNetworkFilterEnabled": {
+          "description": "Flag to indicate whether to enable/disable Virtual Network ACL rules.",
+          "type": "boolean"
+        },
+        "enableAutomaticFailover": {
+          "description": "Enables automatic failover of the write region in the rare event that the region is unavailable due to an outage. Automatic failover will result in a new write region for the account and is chosen based on the failover priorities configured for the account.",
+          "type": "boolean"
+        },
+        "consistencyPolicy": {
+          "description": "The consistency policy for the Cosmos DB database account.",
+          "$ref": "#/definitions/ConsistencyPolicy"
+        },
+        "capabilities": {
+          "type": "array",
+          "description": "List of Cosmos DB capabilities for the account",
+          "items": {
+            "$ref": "#/definitions/Capability"
+          }
+        },
+        "writeLocations": {
+          "type": "array",
+          "readOnly": true,
+          "description": "An array that contains the write location for the Cosmos DB account.",
+          "items": {
+            "$ref": "#/definitions/Location"
+          }
+        },
+        "readLocations": {
+          "type": "array",
+          "readOnly": true,
+          "description": "An array that contains of the read locations enabled for the Cosmos DB account.",
+          "items": {
+            "$ref": "#/definitions/Location"
+          }
+        },
+        "failoverPolicies": {
+          "type": "array",
+          "readOnly": true,
+          "description": "An array that contains the regions ordered by their failover priorities.",
+          "items": {
+            "$ref": "#/definitions/FailoverPolicy"
+          }
+        },
+        "virtualNetworkRules": {
+          "type": "array",
+          "description": "List of Virtual Network ACL rules configured for the Cosmos DB account.",
+          "items": {
+            "$ref": "#/definitions/VirtualNetworkRule"
+          }
+        },
+        "enableMultipleWriteLocations": {
+          "description": "Enables the account to write in multiple locations",
+          "type": "boolean"
+        }
+      }
+    },
+    "DatabaseAccountPatchProperties": {
+      "description": "Properties to update Azure Cosmos DB database accounts.",
+      "type": "object",
+      "properties": {
+        "capabilities": {
+          "type": "array",
+          "description": "List of Cosmos DB capabilities for the account",
+          "ite
```

Comment:
This property is never being populated. It appears the API is returning:
```
		"properties": {
		"id": "seven-day-tables",
		"indexes": [{
			"key": {
				"key": ["aKey"]
			},
			"options": {
				"unique": true
			}
		}, {
```

So because this is keys not key it remains empty?

Reply:
This is not the swagger issue. Now fixed inside our request handler. The returning will be:

"properties": {
		"id": "seven-day-tables",
		"indexes": [{
			"key": {
				"keys": ["aKey"]
			},
			"options": {
				"unique": true
			}
		}, {